README.md

```diff
@@ -317,6 +318,13 @@ Crates tailored for specific boards.
 
 - [`betafpv-f3`](https://github.com/JoshMcguigan/betafpv-f3) - For the BetaFPV F3 drone flight controller
 
+## Component abstraction crates
```

Comment:
@rust-embedded/resources 

Any opinions on adding this new category? Probably worth discussing if we want the crate's contents it in `embedded-hal` itself.

Reply:
I spoke with @japaric (and I think also briefly with @jamesmunns) about this at OxidizeConf. My understanding is `embedded-hal` is intended to be lower level than these sort of "component HALs" and, at least for now, it's best to keep them in their own crates (there's always the option of merging them in the future), and they recommended creating a new section for them.

Happy to discuss what the section should be named though... that's the best i could come up with.

Reply:
Ah, thanks for the context, makes sense. In this case, I just tried to think of what other stuff would eventually fall into this category in the future. I think other typical (Smartphone) SoC sensors are likely candidates, e.g. temperature, barometer, gyro, ambient light, and so on.

Would `sensor` as a category make sense? Or can you think of an example that would fit here that is not a sensor?

Reply:
Some other stuff I think would probably make sense to put under this category:

- [smart-leds-rs](https://github.com/smart-leds-rs/smart-leds)
- [keypad](https://github.com/e-matteson/keypad)
- [embedded-graphics](https://crates.io/crates/embedded-graphics)

Reply:
I updated this section to add `smart-leds-rs` and `embedded-graphics` (moving them from the `no_std` crates section). I think this section from the `smart-leds-rs` docs gets the idea of this category of "component HAL" crate across:

> Use various [component category] in rust. As their protocol greatly differs and for some [...] there are various methods, this crate doesn't implement any specific drivers by itself. It has to be coupled with a device driver that implements the [trait]

...the idea being these crates provide abstraction traits and types those traits are defined in terms of, to provide generic abstractions for a component category, but don't do anything in and of themselves, but need to be combined with a driver crate that implements the trait(s) supplied by these crates.